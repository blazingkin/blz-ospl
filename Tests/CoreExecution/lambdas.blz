:main
    add_two = (x -> x + 2)
    print(add_two(5))

    arr = [1,2,3]
    print(arr.map(( elem -> elem ** 2 )))

end