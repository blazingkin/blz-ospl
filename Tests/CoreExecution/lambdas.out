7
[1, 4, 9]
